.github/actions/setupGitForOSBotify/action.yml

```
@@ -0,0 +1,64 @@
+# This action configures Git for OSBotify.
+#
+# By default, the token it provides is a Github App for Github Authentication.
+# GitHub Apps have higher rate limits.
+#
+# If you only need a local git setup, you can skip that step with SETUP_AS_APP: false
+
+name: Setup Git for OSBotify
+description: Setup Git for OSBotify
+
+inputs:
+  OP_SERVICE_ACCOUNT_TOKEN:
+    description: 1Password service account token
+    required: true
+  OS_BOTIFY_APP_ID:
+    description: Application ID for OS Botify
+    required: false
+  SETUP_AS_APP:
+    description: Should we get an app token for OSBotify?
+    required: false
+    default: 'true'
+
+outputs:
+  # Do not try to use this for committing code. Use `secrets.OS_BOTIFY_COMMIT_TOKEN` instead
+  OS_BOTIFY_API_TOKEN:
+    description: Token to use for GitHub API interactions.
+    value: ${{ steps.generateToken.outputs.token }}
+
+runs:
+  using: composite
+  steps:
+    - name: Install 1Password CLI
+      uses: 1password/install-cli-action@v1
+
+    - name: Download OSBotify GPG key
+      run: op read op://Mobile-Deploy-CI/OSBotify-private-key.asc/OSBotify-private-key.asc --force --out-file ./OSBotify-private-key.asc
```

Even if there are no spaces here, I'd `"` around the name

```suggestion
      run: op read "op://Mobile-Deploy-CI/OSBotify-private-key.asc/OSBotify-private-key.asc" --force --out-file ./OSBotify-private-key.asc
```